toolchain/sem_ir/file.h

```diff
@@ -59,6 +59,11 @@ struct Function : public Printable<Function> {
   // function, in lexical order. The first block is the entry block. This will
   // be empty for declarations that don't have a visible definition.
   llvm::SmallVector<InstBlockId> body_block_ids = {};
+
+  // Given a parameter reference instruction from `param_refs_id` or
+  // `implicit_param_refs_id`, returns the corresponding parameter instruction.
```

Comment:
I'm not sure what the meaning of "parameter instruction" is. Is it specifically a SemIR::Param?

```suggestion
  // Given a parameter reference instruction from `param_refs_id` or
  // `implicit_param_refs_id`, returns the corresponding SemIR::Param's ID,
  // resolving wrapper instructions.
```

Note, if you change to return a pair with `SemIR::Param` (see comments in cpp) I think this is moot.

Reply:
Updated to match new return type.